system_test/aest_hard_fork_SUITE.erl

```diff
@@ -551,16 +551,20 @@ new_node_can_mine_spend_tx_on_old_chain_using_old_protocol(Cfg) ->
     ReceivedBalance = get_balance(new_node3, PubKey1, Cfg),
     ct:log("Balance of node ~p with public key ~p is ~p", [new_node1, PubKey1, ReceivedBalance]),
     ?assertEqual(BalanceToSpend, ReceivedBalance),
+    %% new_node1 has one spend transaction.
+    [#{block_hash := SpendTxBlockHash, block_height := SpendTxHeight, tx := TxInfo}] =
+        get_account_txs(new_node3, PubKey1, Cfg),
```

Comment:
I see type spend_tx is inside function. Ok.

Reply:
This will be changed soon. I'm extracting the HTTP API functions from `aehttp_integration_SUITE.erl`.